lib/bcu/options.rb

```diff
@@ -126,4 +118,78 @@ def self.validate_options(options)
       exit 1
     end
   end
+
+  def self.build_config(use_config_file: true)
+    options = use_config_file ? load_default_options : create_default_options
+
+    options.casks = nil
+    options.install_options = ""
+    options.list_pinned = false
+    options.pin = nil
+    options.unpin = nil
+    options.command = "run"
+
+    options
+  end
+
+  def self.load_default_options
+    config_filename = "#{ENV["HOME"]}/.brew-cu"
+    unless File.exist?(config_filename)
+      odebug "Config file doesn't exist, creating"
+      create_default_config_file config_filename
+    end
```

Comment:
i don't expect here as an user that a normal `brew cu` creates always the config file.

if there is no config then do not create one. 

only if there is an command `brew cu --create-config` i would expect that a config will be created.